change/@ni-nimble-components-876e3564-bb4e-4e2e-a165-f2efd10f9986.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I can't recall if we agreed on a workflow for these PRs that don't have tests, but I'd propose we move it back to Draft once the initial implementation review is approved. That will keep it out of reviewers' view until the tests are added. It looks like it's pretty much ready for that but I'll let @rajsite comment on his status once he looks at the latest response to his feedback

Reply:
Yea it looks like there are a few tests but guessing there will be more around testing out state updates / dynamic changes so yea looks ready to finish adding tests